design/media_types.go

```diff
@@ -31,3 +31,66 @@ var AuthToken = MediaType("application/vnd.authtoken+json", func() {
 		Attribute("token")
 	})
 })
+
+var WorkItem = MediaType("application/vnd.workitem+json", func() {
+	TypeName("WorkItem")
+	Description("ALM Work Item")
+	Attribute("id", String, "unique id per installation")
```

Comment:
Oh no, I just question the use of an installation wide unique ID as it is not copyable to another installation. That's all.